packages/wallets/src/hooks/useWalletsList.ts

```diff
@@ -95,10 +109,21 @@ export default function useWalletsList(search?: string): {
       return undefined;
     }
 
-    const baseWallets = wallets.filter((wallet) => ![WalletType.CAT, WalletType.POOLING_WALLET].includes(wallet.type));
+    const baseWallets = wallets.filter((wallet: Wallet) => ![WalletType.CAT, WalletType.POOLING_WALLET].includes(wallet.type));
```

Comment:
No change needed now, but just to let you know that there will also be DID and NFT wallets to consider here.